### ISSUE TITLE
Error when running schnetpack_qm9.py in wacsf mode

### ISSUE THREAD

**Issue by OMalenfantThuot** [atomistic-machine-learning/schnetpack#112]
Hi,
when running the schnetpack_qm9.py script this way:
`schnetpack_qm9.py train wacsf qm9.db model2/ --split 100 100`

I got the output:
```
INFO:root:Random state initialized with seed 3298687774
INFO:root:QM9 will be loaded...
INFO:schnetpack.data.atoms:The dataset has already been downloaded and stored at qm9.db
INFO:root:create splits...
INFO:root:load data...
INFO:root:calculate statistics...
INFO:schnetpack.data.loader:statistics will be calculated...
Traceback (most recent call last):
  File "/home/oliviermt/miniconda3/bin/schnetpack_qm9.py", line 78, in <module>
    representation = get_representation(args, train_loader=train_loader)
  File "/home/oliviermt/miniconda3/lib/python3.6/site-packages/scripts/script_utils/model.py", line 12, in get_representation
    if args.cutoff_function == "hard":
AttributeError: 'Namespace' object has no attribute 'cutoff_function'
```
And there is no error when using the schnet mode.

In file src/scripts/script_utils/model.py, the `# build cutoff module` block should be inside the `if args.model == "schnet"` block if it is not used in wacsf.

I corrected it [here](https://github.com/OMalenfantThuot/schnetpack/blob/devel/src/scripts/script_utils/model.py) and commented the old block and it seems to be working.

Thanks a lot!

**Comment by Stefaanhess** [comment]
Hi @OMalenfantThuot,

You are right! I included your solution in #113 .

**Comment by OMalenfantThuot** [comment]
Thanks!
